service/controller/v12/resource_set.go

```diff
@@ -353,6 +367,7 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		containerURLResource,
 		encryptionkeyResource,
 		blobObjectResource,
+		ingressPIPNameResource,
```

Comment:
`deploymentResource` creates the given public IP so this resource must be run right before it.